packages/deployments/contracts/tasks/connector/sendAxelar.ts

```diff
@@ -0,0 +1,60 @@
+import { Contract } from "ethers";
+import { task } from "hardhat/config";
+
+import {
+  Env,
+  getConnectorName,
+  getDeploymentName,
+  getMessagingProtocolConfig,
+  mustGetEnv,
+  ProtocolNetwork,
+} from "../../src/utils";
+import { calculateAxelarBridgeFee, chainIdToDomain } from "@connext/nxtp-utils";
+
+type TaskArgs = {
+  data?: string;
+  env?: Env;
+  networkType?: ProtocolNetwork;
+};
+
+export default task("connector-send-axelar", "Call `Connector.send()` to distribute outbound root via axelar connector")
```

Comment:
this task should exist as a special case of `send-connector` based on chain, should not exist as a standalone task